front/components/AlgoliaSearch.js

```diff
@@ -135,9 +147,21 @@ export default class AlgoliaSearch extends React.Component {
             return `/emmanuel-macron/desintox/${hit.slug}`;
         }
 
+        if ('event' === hit.type) {
+            return `/evenements/${hit.uuid}/${hit.slug}`;
```

Comment:
It's called [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).